### PR TITLE
Grid component

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: Components/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    </div>
+
+    <div class="grid-container four-item-grid">
+      <div class="grid-item">Item 1</div>
+      <div class="grid-item">Item 2</div>
+      <div class="grid-item">Item 3</div>
+      <div class="grid-item">Item 4</div>
+    </div>
+  </body>
+</html>
```
Comment (b0llu) [inline Components/grid/grid.html:29]: Well organised and well named html great work

**Review by b0llu — APPROVED** [review]
Nothing much to suggest Great work on writing clean code and semantic html

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Everything Looks Good Nice Work

**Review by nayyyhaa — APPROVED** [review]
Proper naming + code is well structured. you can merge the PR. 👍

**Pull request merged**